ui/js/models/resultset.js

```diff
@@ -120,11 +120,11 @@ treeherder.factory('ThResultSetModel', ['$http', '$location',
             },
             getResultSet: function (repoName, pk) {
                 return $http.get(
-                    thUrl.getProjectUrl("/resultset/" + pk + "/", repoName)
+                    getProjectUrl("/resultset/" + pk + "/")
```

Comment:
Could you switch this and the other modified lines in this file to a template literal? :-)